charts/pravega-operator/values.yaml

```diff
@@ -0,0 +1,7 @@
+image:
+  repository: pravega/pravega-operator
+  tag: 0.1.0
+  pullPolicy: Always
+
+watch:
```

Comment:
How about we not expose this `WATCH_NAMESPACE` facility for now.  It would imply a different RBAC manifest (e.g. `Role` rather than `ClusterRole`).

Reply:
I've added multiple RBAC templates that trigger depending if the Watch namespace is set or not